frontend/src/app/main/ui/workspace/tokens/management/create/form.cljs

```diff
@@ -848,8 +850,9 @@ custom-input-token-value-props: Custom props passed to the custom-input-token-va
            :type "button"}])]
     [:*
      [:> input-tokens-value*
-      {:placeholder (tr "workspace.tokens.token-font-family-value-enter")
-       :label (tr "workspace.tokens.token-font-family-value")
+      {:placeholder (or placeholder (tr "workspace.tokens.token-font-family-value-enter"))
```

Comment:
I don't understand this behaviour well. Does this input sometimes not have a visible label?

If so, the name of the component is unclear to me because it does not seem to be an input, and I would not expect this component to use aria-label.

I suggest being explicit about the UI component (e.g. `font-picker-select*`).